spek-runtime/src/commonMain/kotlin/org/spekframework/spek2/runtime/Collectors.kt

```diff
@@ -71,11 +73,12 @@ class Collector(
 
     }
 
-    override fun test(description: String, skip: Skip, body: TestBody.() -> Unit) {
+    override fun test(description: String, skip: Skip, timeout: Long, body: TestBody.() -> Unit) {
         val test = TestScopeImpl(
             idFor(description),
             root.path.resolve(description),
             root,
+            defaultTimeout,
```

Comment:
you want to use the argument here instead

Reply:
yep, already fixed in master.

Reply:
so I have this: 
```class ResponseTest : Spek({

    setArchTaskExecutor()

    timeout(15000)
```
but that doesn't work. How to I set it?

Reply:
`defaultTimeout = xxx`